.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -1262,43 +1262,63 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
             var ret = new DateTimeResolutionResult();
 
             var er = this.config.DateExtractor.Extract(text, referenceDate);
-            if (er.Count < 2)
-            {
-                er = this.config.DateExtractor.Extract(this.config.TokenBeforeDate + text, referenceDate);
-                if (er.Count < 2)
-                {
-                    return ret;
```

Comment:
Shouldn't the attempt to extract "now" be here, and if nothing is extracted, then return ret?